### ISSUE TITLE
HA logs: AttributeError: 'UUID' object has no attribute 'replace'

### ISSUE THREAD

**Issue by jonoberheide** [AngellusMortis/pyunifiprotect#334]
Been seeing this exception spamming in the logs every few minutes. Latest HA version (2023.12.4), although I've been seeing it since adding the UniFi Protect HA integration.

Nothing unusual about my Protect setup that I can think of, other than using stacked NVRs which may be less common. Happy to debug further.

```
Logger: pyunifiprotect.websocket
Source: /usr/local/lib/python3.11/site-packages/pyunifiprotect/websocket.py:84
First occurred: December 28, 2023 at 10:36:47 PM (175 occurrences)
Last logged: 11:07:24 AM

Error processing websocket message
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/pyunifiprotect/websocket.py", line 82, in _process_message
    sub(msg)
  File "/usr/local/lib/python3.11/site-packages/pyunifiprotect/api.py", line 694, in _process_ws_message
    processed_message = self.bootstrap.process_ws_packet(
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/pyunifiprotect/data/bootstrap.py", line 573, in process_ws_packet
    return self._process_nvr_update(packet, data, ignore_stats)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/pyunifiprotect/data/bootstrap.py", line 447, in _process_nvr_update
    self.nvr = self.nvr.update_from_dict(deepcopy(data))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/pyunifiprotect/data/base.py", line 546, in update_from_dict
    setattr(self, key, convert_unifi_data(data[key], self.__fields__[key]))
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/pyunifiprotect/utils.py", line 238, in convert_unifi_data
    return type_(value)
           ^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/uuid.py", line 175, in __init__
    hex = hex.replace('urn:', '').replace('uuid:', '')
          ^^^^^^^^^^^
AttributeError: 'UUID' object has no attribute 'replace'
```

**Comment by jonoberheide** [comment]
I should note that everything seems to function ok despite the log spam.

**Comment by jonoberheide** [comment]
If I'm reading that function and exception properly, it looks like the `value` is already uuid.UUID type. So it's calling UUID() on a uuid object instead of a string.

**Comment by AngellusMortis** [comment]
Fixed in 4.22.4
